File: scala-spring/accounts-command-side-service/src/main/scala/net/chrisrichardson/eventstore/examples/bank/web/main/AccountsCommandSideServiceMain.scala
package net.chrisrichardson.eventstore.examples.bank.web.main

import net.chrisrichardson.eventstore.examples.bank.web.AccountsCommandSideServiceConfiguration
import org.springframework.boot.SpringApplication

object AccountsCommandSideServiceMain  {

  def main(args: Array[String]) : Unit = SpringApplication.run(classOf[AccountsCommandSideServiceConfiguration], args :_ *)

}

File: java-spring/accounts-command-side-service/src/main/java/net/chrisrichardson/eventstore/javaexamples/banking/web/main/AccountsCommandSideServiceMain.java
package net.chrisrichardson.eventstore.javaexamples.banking.web.main;

import net.chrisrichardson.eventstore.javaexamples.banking.web.AccountsCommandSideServiceConfiguration;
import org.springframework.boot.SpringApplication;

public class AccountsCommandSideServiceMain {

  public static void main(String[] args) {
    SpringApplication.run(AccountsCommandSideServiceConfiguration.class, args);
  }
}
